src/misc/ListModel.ts

```diff
@@ -328,8 +328,17 @@ export class ListModel<ElementType extends ListElement> {
 	}
 
 	/** An element was added to the selection. If multiselect was not on, add previous single selection and newly added selected item to the selection. */
-	onSingleInclusiveSelection(item: ElementType): void {
+	onSingleInclusiveSelection(item: ElementType, clearSelectionOnMultiSelectStart?: boolean): void {
+		// If isn't in MultiSelect, we discard all previous items
+		// and start a new set of selected items in MultiSelect mode
+		// we do it only if the user is on singleColumnMode, otherwise
+		// we want to keep the original behavior
```

Comment:
Also, I feel like "original behavior" can be confusing in the future (what "original behavior" was gets lost to time). Would describing the behaviors here make a too-long comment? Or maybe just say, "We do it only if the user is in singleColumnMode, because there are different expected behaviors there."?

Reply:
I agree, in the future, maybe not so distant, it will be confusing for those who will maintain it